### PR TITLE
WIP - initial loading coach remotely

### PR DESCRIPTION
**Pull request by pandafulmanda**

### CODE REVIEW

File: src/scripts/config.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 (function () {
   'use strict';
 
+  var coachBasePath = '//openstax.github.io/concept-coach/';
```
Comment (philschatz) [inline src/scripts/config.js]: - just verifying that this URL is for testing purposes only, right?
- will this URL be loaded for everyone that goes to CNX or just those in a CC class?

Comment (pandafulmanda) [inline src/scripts/config.js]: * yes
* i will need to check, and we can look into doing that, but i think that's secondary to getting coach to not need a hash update in cnx right?

Comment (philschatz) [inline src/scripts/config.js]: I believe CNX has a policy of not depending on other sites (except google analytics and I guess now zendesk) but others might be able to chime in.

**Comment by edwoodward** [comment]
@pandafulmanda Is this still being worked on?

**Comment by pandafulmanda** [comment]
we will eventually get back to this, but we can close this for now

**Pull request closed without merging**